video-course/src/app/app.component.html

```diff
@@ -1,3 +1,11 @@
-<vc-header></vc-header>
-<vc-courses></vc-courses>
-<vc-footer></vc-footer>
\ No newline at end of file
+<div *ngIf='true' class='home-page'>
+  <vc-header isDisplayLogin='true'></vc-header>
+  <vc-courses></vc-courses>
+  <vc-footer></vc-footer>
+</div>
+<div *ngIf='false' class='login-page'>
+  <vc-header isDisplayLogin='false' ></vc-header>
+  <vc-login-page></vc-login-page>
+  <vc-footer></vc-footer>
+</div>
+
```

Comment:
I guess this is temporally before routing implementation.
However, try not to duplicate code.
1. Header and footer are kinda the same (only breadcroms are hidden)
so, you can toggle between :
<vc-courses> and <vc-login-page></vc-login-page>

2. isDisplayLogin='true' , is this to display: login / logout ? if yes, display this base on "is the user logged" if you don't have this implemented yet, just display both options.